gordo_components/builder/build_model.py

```diff
@@ -300,7 +309,7 @@ def provide_saved_model(
                 f"{model_register_dir}."
             )
     model, metadata = build_model(
-        model_config=model_config, data_config=data_config, metadata=metadata
+        name=name, model_config=model_config, data_config=data_config, metadata=metadata
```

Comment:
I will put it in the provide_saved_model, before the calculate key thing. Not further out, because I think it is nice that there some predictability in that if you call provided_saved_model, without using the cli, you can still construct a working endpoint (by having the model name).